### PR TITLE
PMM-10493 Fix portal force disconnect

### PR DESCRIPTION
**Pull request by matejkubinec**
**What this PR does / why we need it**:

Fixes portal disconnection as an error from BE is expected in a force disconnection flow, a successful message should be shown instead of an error notification.

Values used for connecting are cleared after successful connection. (Connected pmm uses a different view)

**Which issue(s) this PR fixes**:

Fixes PMM-10493

### CODE REVIEW

File: public/app/percona/settings/components/Platform/Connected/Connected.tsx [hunk 1]
```diff
@@ -46,13 +48,21 @@ export const Connected: FC = () => {
...
-      setDisconnecting(false);
-      dispatch(fetchServerInfoAction());
-      dispatch(fetchSettingsAction());
     } catch (e) {
-      logger.error(e);
-      setDisconnecting(false);
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      const error = e as AxiosError<ForceDisconnectErrorBody>;
+
+      if (error.response?.data?.code === 9) {
```
Comment (fabio-silva) [inline public/app/percona/settings/components/Platform/Connected/Connected.tsx:55]: Out of curiosity, what's the meaning of `9` here?

Comment (matejkubinec) [reply public/app/percona/settings/components/Platform/Connected/Connected.tsx:55]: I need to check with someone on the BE if it has a specific meaning. I might change the condition to check the error message text instead.

Comment (matejkubinec) [reply public/app/percona/settings/components/Platform/Connected/Connected.tsx:55]: @fabio-silva Just checked with Jiri, each code corresponds to a specific error message (and vice versa), so I think we should be fine here using the code to differentiate the portal disconnect error. (I'll add a comment to explain it in code)

Comment (fabio-silva) [reply public/app/percona/settings/components/Platform/Connected/Connected.tsx]: got it 👍

**Review by fabio-silva — DISMISSED** [review]

**Review by solovevayaroslavna — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

**Review by solovevayaroslavna — APPROVED** [review]

**Pull request merged**